tests/versioned/apollo-server/transaction-naming.test.js

```diff
@@ -51,8 +52,19 @@ tap.test('apollo-server', (t) => {
     serverUrl = null
     helper = null
 
-    done()
+    clearCachedModules(['express', 'apollo-server'], () => {
+      done()
+    })
   })
 
   createTransactionTests(t)
 })
+
+function clearCachedModules(modules, callback) {
```

Comment:
Move this func to a utils file since its used in a couple places now?

Reply:
If you move the function, the resolve path will be different and it won't clear out what you are trying to clear out. There's probably a way to do generically but I'm unsure off the top of my head.

Reply:
Ah ok, 👍